.github/workflows/test.yaml

```diff
@@ -0,0 +1,88 @@
+name: Tests stage - unit test for sniffs
+
+on:
+    pull_request:
+        branches: [ master, develop ]
```

Comment:
Just documenting that this will work based on: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags.

I was originally looking [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request) and did not see the branches documented.